testing/unit_tests/t_utils.py

```diff
@@ -20,6 +20,14 @@ def download_data_testing_test_files():
     download_dataset("data_testing")
 
 
+@pytest.fixture(scope='session')
+def download_multi_data():
+    """
+    This Pytest fixture DOWNLOAD all the test data set REQUIRED for the multi-session, multi-contrast related unite
+    testing.
+    """
+    download_dataset("data_multi_testing")
```

Comment:
New pytest fixture for downloading multi-session/contrast related data, a key dependent fixture for all multi-session/contrasts related unit tests.

Reply:
Being split into clean up PR.